<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvarList</class>
 <widget class="QDialog" name="InvarList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Invariant List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Invariant List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="itemList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>780</width>
     <height>270</height>
    </rect>
   </property>
  </widget>
  <widget class="LineEditRU" name="ruEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="engLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ruLbl">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="typeLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Invariant Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>330</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findBttn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBttn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="LineEditEN" name="enEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newBttn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBttn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="declLbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Takes Declension</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="declBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>330</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditRU</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineeditru</header>
  </customwidget>
  <customwidget>
   <class>LineEditEN</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineediten</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>itemList</tabstop>
  <tabstop>enEdit</tabstop>
  <tabstop>ruEdit</tabstop>
  <tabstop>typeBox</tabstop>
  <tabstop>declBox</tabstop>
  <tabstop>quitBttn</tabstop>
  <tabstop>findBttn</tabstop>
  <tabstop>deleteBttn</tabstop>
  <tabstop>newBttn</tabstop>
  <tabstop>addBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
